client/src/js/auth-helpers.js

```diff
@@ -0,0 +1,103 @@
+import { h, Component, render } from 'preact';
+import Cookies from 'universal-cookie';
+import { route } from 'preact-router';
+import decode from 'jwt-decode';
+import { Menu } from 'preact-material-components';
+import {API_SERVER} from "../../config";
+import axios from "axios/index";
+
+const tokenCookie = 'token';
+
+export const logout = () => {
+	deleteTokenCookie();
+	route('/',true);
+}
+
+export const deleteTokenCookie = () => {
+	const cookies = new Cookies();
+	cookies.remove(tokenCookie);
+}
+
+export const setTokenCookie = (val) => {
+	const cookies = new Cookies();
+	cookies.set(tokenCookie, val, { path: '/' });
```

Comment:
we should set these additional options to improve security:
- expires (Date): absolute expiration date for the cookie - limit it to 24hrs for now to match server
- secure (boolean): Is only accessible through HTTPS?  *I'm not sure if this will cause problems with `localhost` domain but I think not*
- httpOnly (boolean): Is only the server can access the cookie? - we probably want this too which means we need to think through how to handle access to user information for other routes; probably simply pinging the server with cookie to confirm okay to proceed else send user to sign-in page.

This link gives a good overview of why we should set these options: https://stormpath.com/blog/where-to-store-your-jwts-cookies-vs-html5-web-storage#so-whats-the-difference